addons/storyshots/src/index.js

```diff
@@ -21,6 +27,16 @@ const hasDependency = name =>
   (pkg.devDependencies && pkg.devDependencies[name]) ||
   (pkg.dependencies && pkg.dependencies[name]);
 
+function patchStoriesOfFunction(storybookImpl) {
```

Comment:
I think it's a pretty transparent change that could be useful for other purposes.